tests/unit/onyxTest.ts

```diff
@@ -1725,5 +1725,87 @@ describe('Onyx', () => {
                     });
             });
         });
+
+        describe('setCollection', () => {
+            it('should replace all existing collection members with new values and remove old ones', async () => {
+                let result: OnyxCollection<unknown>;
+                const routeA = `${ONYX_KEYS.COLLECTION.ROUTES}A`;
+                const routeB = `${ONYX_KEYS.COLLECTION.ROUTES}B`;
+                const routeB1 = `${ONYX_KEYS.COLLECTION.ROUTES}B1`;
```

Comment:
NAB why call this B1 and not C (and make the next one D)?

Reply:
@iwiznia the intention here was to showcase that the key (`routeB1`) might start with a similar value as the other key (`routeB`) but it would be properly filtered out in the `setCollection`